tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_providers {
+    aws  = ">= 2.31.0"
+  }
+}
+
+provider "aws" {
+  region  = var.region
+  profile = var.profile
+}
+
+resource "aws_db_subnet_group" "default" {
+  name       = var.aws_db_subnet_group
+  subnet_ids = var.subnets
+}
+
+resource "aws_security_group" "rds_cluster_access" {
+  name_prefix   = var.security_group_name
+  vpc_id        = var.vpc_id
+}
+
+resource "aws_secretsmanager_secret" "rds_login" {
+  name_prefix = "cumulus_rds_db_cluster"
+}
+
+resource "aws_secretsmanager_secret_version" "rds_login" {
+  secret_id     = aws_secretsmanager_secret.rds_login.id
+  secret_string = jsonencode({
+    username            = var.db_username
```

Comment:
I might be dense, but where do the initial `db_username` and `db_password` values come from? Oh, nevermind, looks like they are whatever you want and used below when the cluster is setup